examples/systemToParaTeleportForeignAssets.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetsTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { AssetsTransferApi, constructApiPromise } from '../src';
+import { TxResult } from '../src/types';
+import { GREEN, PURPLE, RESET } from './colors';
+
+/**
+ * In this example we are creating a call to send foreign asset '{"parents":"1","interior":{"X2":[{"Parachain":"2125"},{"GeneralIndex":"0"}]}}'
```

Comment:
Here I would clearly mention it will be a teleport because we are transferring a token native to the destination chain.